.gitlab-ci.yml

```diff
@@ -11,3 +11,5 @@ push:
   stage: push
   script:
     - docker push gpii/version-updater
+  only:
+    - master@gpii-ops/gpii-infra
```

Comment:
shouldn't this say `master@gpii-ops/gpii-version-updater`?

Reply:
Oops.